src/System.Net.HttpListener/src/System/Net/HttpListenerResponse.cs

```diff
@@ -61,22 +61,15 @@ public void SetCookie(Cookie cookie)
             {
                 throw new ArgumentNullException(nameof(cookie));
             }
-            bool added = false;
-            try
-            {
-                Cookies.Add(cookie);
-                added = true;
-            }
-            catch (CookieException)
```

Comment:
Previously this was eating all CookieExceptions.  Is that not being done on desktop?